packages/@uppy/companion/src/server/emitter/redis-emitter.js

```diff
@@ -32,18 +62,27 @@ module.exports = (redisUrl, redisPubSubScope) => {
    * @param {object} message the message to pass along with the event
    */
   function emit (eventName, message) {
-    return nrp.emit(eventName, message || {})
+    if (eventName === 'error') {
```

Comment:
AFAIK our app doesn't ever emit the error event, and I'm not sure what's the use case of this.

Reply:
Backward compat? I'm fine with removing it